Dockerfile

```diff
@@ -6,23 +6,26 @@ USER root
 
 RUN apt-get update && apt-get install -y \
     # GCC required to resolve error during JupyterLab installation: psutil could not be installed from sources because gcc is not installed.
-    gcc curl \
+    gcc curl unzip\
     && rm -rf /var/lib/apt/lists/*
 
-# TODO: using Gradle to build the jar
-# Install jars to support delta lake spark operations
-ENV HADOOP_AWS_VER=3.3.4
-RUN curl -O https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/${HADOOP_AWS_VER}/hadoop-aws-${HADOOP_AWS_VER}.jar \
-    && mv hadoop-aws-${HADOOP_AWS_VER}.jar /opt/bitnami/spark/jars
+# Install Gradle
+ENV GRADLE_VERSION=8.7
+RUN curl -L https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip -o gradle-bin.zip \
+    && unzip gradle-bin.zip -d /opt \
+    && rm gradle-bin.zip \
+    && ln -s /opt/gradle-${GRADLE_VERSION}/bin/gradle /usr/bin/gradle
```

Comment:
You should `gradle init` the repo and then use the gradle wrapper file here to install the jars

Reply:
Check out any of the java services for examples

Reply:
run gradle init locally? or with dockerfile? I just need to download the jars not turning this to a java project.

Reply:
run gradle init in the repo locally, that'll set up gradle correctly with a wrapper file

Reply:
If you're using the wrapper you don't need to install gradle

Reply:
👍